apisix/schema_def.lua

```diff
@@ -722,6 +731,13 @@ _M.ssl = {
     type = "object",
     properties = {
         id = id_schema,
+        type = {
+            description = "ssl certificate type, " ..
+                            "0 to server certificate, " ..
+                            "1 to client certificate for upstream",
+            type = "integer",
+            enum = {0, 1}
```

Comment:
Please use enum string like `server` & `client`, thanks!

Reply:
Better to provide a default value, so we don't need to handle an unset situation.